rocAL/include/loaders/loader_module.h

```diff
@@ -53,8 +53,8 @@ class LoaderModule {
     virtual Timing timing() = 0;                    // Returns timing info
     virtual std::vector<std::string> get_id() = 0;  // returns the id of the last batch of images/frames loaded
     virtual void start_loading() = 0;               // starts internal loading thread
-    virtual decoded_image_info get_decode_image_info() = 0;
-    virtual crop_image_info get_crop_image_info() = 0;
+    virtual DecodedDataInfo get_decode_data_info() = 0;
+    virtual crop_image_info get_crop_image_info() { return {}; }
```

Comment:
why is this changed?

Reply:
Changed from pure virtual to virtual since its not used in the Audio. This API is only related to image.